### ISSUE TITLE
Stay at the current workspace after moving an app to another workspace

### ISSUE THREAD

**Issue by ghost** [WayfireWM/wayfire#571]
I would like to be able to stay in my current workspace when I use my vswitch.binding_win_* keys.

```
[vswitch]
# Move the focused window with the same key-bindings, but add Shift.
binding_win_right = <ctrl> <super> <shift> KEY_RIGHT
```

1. Open an app in workspace 1
2. Use `<ctrl> <super> <shift> KEY_RIGHT`
3. App window moves to workspace 2
4. I stay at workspace 1

Swaywm example

```
# move app to workspace 2, stay at workspace 1
bindsym $mod+Shift+2 move container to workspace 2

# move app to workspace 2, focus workspace 2
bindsym $mod+Shift+2 move container to workspace 2, workspace 2
```

**Comment by alexherbo2** [comment]
Having an IPC for such composability could be nice.

**Comment by ammen99** [comment]
> Having an IPC for such composability could be nice.

I fail to see how IPC is relevant, this is a functionality a plugin can easily provide.

**Comment by alexherbo2** [comment]
I find the way of exposing features in Wayfire (a functionality to a binding) less expressive than the other way around that comes with Sway.

**Comment by SnooSnoo-ui** [comment]
I think C++ plugins are very nice to have but there's people like me who couldn't write one to save their lives or fear introducing bugs. For everything that needs performance or can't be done with IPC. With IPC I believe more hobbyists are willing to try their luck writing their own functionality. It could help the growth of the project when they share on GitHub, reddit and other places. It's nice to be able to bind a script using IPC, write a daemon or other combinations. I just wouldn't know how much of that is used to overcome tiling limitations and how much value it has for stacking. Would have to check the most popular scripts for other stacking WM and see how they used the IPC or custom scripts.

**Comment by ammen99** [comment]
Fixed in #1199
